index.html

```diff
@@ -0,0 +1,300 @@
+                    <span class="text-secondary align-self-end ml-3">
+                      © 2021 NEWSWEEK DIGITAL LLC
+                      </span>
+                </div>
+                <div>
+                  <i class="fab fa-facebook-square fa-2x"></i>
+                  <i class="fab fa-twitter fa-2x"></i>
+                  <i class="fab fa-instagram fa-2x"></i>
+                  <i class="fab fa-tumblr fa-2x"></i>
+                  <i class="fab fa-linkedin-in fa-2x"></i>
```

Comment:
- [ ] Use semantic HTML for links. `<a>` anchor tag

Reply:
done